modules/apps/foundation/portal-security-sso/portal-security-sso-openid-connect-api/src/main/java/com/liferay/portal/security/sso/openid/connect/OpenIdConnectSession.java

```diff
@@ -0,0 +1,7 @@
+package com.liferay.portal.security.sso.openid.connect;
+
+public interface OpenIdConnectSession {
+	
+	public String getAccessTokenString();
```

Comment:
This should contain other methods to access general state information that is part of the OIDC spec. Refresh token, login time, etc

Reply:
Where is the spec you're referring to? I found http://openid.net/specs/openid-connect-core-1_0.html but I'm not seeing a definitive list of state attributes.